funcr/funcr.go

```diff
@@ -500,6 +501,21 @@ func (f Formatter) prettyWithFlags(value interface{}, flags uint32, depth int) s
 		}
 		return buf.String()
 	case reflect.Slice, reflect.Array:
+		// If this is outputing as JSON make sure this isn't really a json.RawMessage.
+		// If so just emit "as-is" and don't pretty it as that will just print
+		// it as [X,Y,Z,...] which isn't terribly useful vs the string form you really want.
+		if f.outputFormat == outputJSON {
+			if rm, ok := value.(json.RawMessage); ok {
+				// If it's empty make sure we emit an empty value as the array style would below.
+				if len(rm) > 0 {
+					buf.Write(rm)
+				} else {
+					buf.WriteByte('{')
```

Comment:
I wonder if there's a schema problem here - if the consumer of the log expects an int or string or list, and you give them an object, will it cause a problem?  I guess it's a super corner-case, but why not put "null" here instead?

Reply:
Ahh...true. Let me do that